pkg/cmd/dev/doctor.go

```diff
@@ -138,6 +181,9 @@ Please add one of the following to your %s/.bazelrc.user:`, workspace)
 	}
 
 	if success {
+		if err := writeDoctorStatus(ctx, d.exec); err != nil {
```

Comment:
At this point we already have the `workspace` variable which contains the path to the workdir. So you can just pass `workspace` as an argument.